src/topology/algebra/module.lean

```diff
@@ -51,11 +51,22 @@ class topological_module (α : Type u) (β : Type v)
   [module α β]
   extends topological_semimodule α β : Prop
 
+
+/- Note[topological vector space definition]:
```

Comment:
I think this should rather be in the implementation notes section of the file header.

Reply:
Done.